app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/multipledisposals/country_of_residence.scala.html

```diff
@@ -0,0 +1,68 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.RequestWithSessionData
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.triage
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.address.Country
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.views.components.AutoCompleteType
+
+@this(
+        mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
+        countryCode: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.country_code,
+        submitButton:  uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.submit_button,
+        formWrapper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+        errorSummary: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.error_summary,
+        backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link,
+        caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption,
+        returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link
+)
+
+@(form: Form[Country], backLink: Call)(implicit request: RequestWithSessionData[_], messages:Messages, appConfig: ViewConfig)
+
+@key = @{"countryCode"}
+@title = @{messages("countryCode.title")}
+@hasErrors = @{form.hasErrors || form.hasGlobalErrors}
+
+@mainTemplate(title = title, hasErrors = hasErrors, userType = request.userType, withSignOutLink = true) {
+
+    @backLinkComponent(backLink)
+
+    @if(hasErrors) {
+        @errorSummary(form)
+    }
+
+    @caption(messages("multiple-disposals.triage.caption"))
+
+    @formWrapper(triage.routes.MultipleDisposalsTriageController.countryOfResidenceSubmit(), 'novalidate -> "novalidate") {
+
+        @countryCode(
+            countryCodeToCountryName = Country.countryCodeToCountryName,
+            selected = form.data.get(key),
+            label = title,
+            hasErrors = form.error(key).nonEmpty,
+            errorKey = form.error(key).map(e => e.message),
+            autoComplete = AutoCompleteType.Country,
+            helpText = Some(Html(messages(s"countryCode.helpText"))),
```

Comment:
it'd be better to scope the help text message for this page a bit more narrowly - the help text isn't for all pages which have something to do with country codes.